src/Resources/config/config.yaml

```diff
@@ -70,6 +70,11 @@ sylius_ui:
                 pay_pal_payment_method:
                     template: '@SyliusPayPalPlugin/javascripts.html.twig'
                     priority: -100
+        sylius.admin.order.show.payment_content:
+            blocks:
+                pay_pal_refunded:
+                    template: '@SyliusPayPalPlugin/Admin/Order/Show/Payment/_refundedPayPal.html.twig'
+                    priority: 9 # After refund button
```

Comment:
```patch
--- # After refund button
+++ # After "refund_transition"
```

Refund button and refunded message won't show up at the same time, so it makes more sense to mention block name instead of html element.